src/movepick.cpp

```diff
@@ -177,6 +177,15 @@ void MovePicker::score<QUIETS>() {
   Color c = pos.side_to_move();
 
   for (auto& m : *this)
+#ifdef ANTI
+      if (pos.is_anti() && (pos.attackers_to(to_sq(m)) & pos.pieces(~c)))
```

Comment:
While this is an improvement, it isn't accurate because the piece may already be attacked, in which case it may be blocking the "to" square.

Also I would be surprised if the compiler did loop unrolling here; `(pos.attackers_to(to_sq(m)) & pos.pieces(~c))` is not easy to statically evaluate, although thankfully it benefits from `Position::attacks_from<PAWN>` cached data.  I suppose I'll try some ideas to mitigate impact, such as creating an `attackers_to<Color>` and removing `& pos.pieces(~c)`.